build-aux/generate.mk

```diff
@@ -504,6 +505,16 @@ $(OSS_HOME)/OPENSOURCE.md: $(tools/go-mkopensource) $(tools/py-mkopensource) $(O
 		{ sed 's/^---$$//' $(OSS_HOME)/build-aux/pip-show.txt; echo; } | $(tools/py-mkopensource); \
 	} > $@
 
+$(OSS_HOME)/LICENSES.md: $(tools/go-mkopensource) $(tools/py-mkopensource) $(OSS_HOME)/build-aux/go-version.txt $(OSS_HOME)/build-aux/pip-show.txt
+	$(MAKE) $(OSS_HOME)/build-aux/go$$(cat $(OSS_HOME)/build-aux/go-version.txt).src.tar.gz
+	set -e; { \
+		cd $(OSS_HOME); \
+		echo -e "Emissary-ingress Go code incorporates Free and Open Source software under the following licenses:\n"; \
+		$(tools/go-mkopensource) --output-format=txt --package=mod --output-type=json --gotar=build-aux/go$$(cat $(OSS_HOME)/build-aux/go-version.txt).src.tar.gz | jq -r '.licenseInfo | to_entries | .[] | "* [" + .key + "](" + .value + ")"' | sed -e 's/\[\([^]]*\)]()/\1/'; \
+		echo -e "\n\nEmissary-ingress Python code incorporates Free and Open Source software under the following licenses:\n"; \
+		{ sed 's/^---$$//' $(OSS_HOME)/build-aux/pip-show.txt; echo; } | $(tools/py-mkopensource) --output-type=json | jq -r '.licenseInfo | to_entries | .[] | "* [" + .key + "](" + .value + ")"' | sed -e 's/\[\([^]]*\)]()/\1/'; \
+	} > $@
+
```

Comment:
Yes, I think it would be easier to maintain. I can pull out this one and also `$(OSS_HOME)/OPENSOURCE.md`

Would `build-aux/license-info/` be a good location for license generation scripts?

Reply:
@kflynn I'm doing these changes as part of https://github.com/emissary-ingress/emissary/pull/4027
For this PR I'll leave it as so I don't have to cherry pick from the other PR.

Let me know if you have any concerns